### ISSUE TITLE
No Sophomorix-Schema-Version in AD found

### ISSUE THREAD

**Issue by zefanja** [linuxmuster/sophomorix4#33]
When I upgrade sophomorix-samba I get this message in the post-install section:

```
Testing for sophomorix schema update
   WARNING: No Sophomorix-Schema-Version in AD found: Skipping updates
```
Is there anything I should do or is it normal?

**Comment by jeffbeck** [comment]
That should not happen on a recently new version that you upgrade FROM.

The update process is documented here:

https://github.com/linuxmuster/sophomorix4/wiki/Schema

Can you set back your testing environment before the update?

**Comment by jeffbeck** [comment]
closed in 3.60.10 see issue 35
